packages/block-library/src/embed/core-embeds.js

```diff
@@ -25,6 +25,15 @@ import { __ } from '@wordpress/i18n';
 import { createBlock } from '@wordpress/blocks';
 
 export const common = [
+	{
```

Comment:
What is the reason it needs to be its own block? Can't be it included inside the default embed block?

Reply:
No, we have no way of knowing that the url has come from a paste, other than having a specific paste handler block.